doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
 
-*-s*[date], *--startday*[=date]::
-  Print  events  and appointments from 'date' and exit.  If no 'date' is given,
-  the current day is considered. Equivalent to *-Q --filter-type cal --from
-  <date>*.
+*-s*['date'], *--startday*[='date']::
+  Print events and appointments from the optional 'date'; default is the
+  current day. Equivalent to *-Q --filter-type cal --from* 'date'.
 
-*-S*<regex>, *--search*=<regex>::
-  When used with the *-a*, *-d*, *-r*, *-s*, or *-t* flag, print only the items
-  having a description that matches the given regular expression. Equivalent to
-  *-Q --filter-pattern <regex>*.
+*-S* 'regex', *--search* 'regex'::
+  When used with any of *-a*, *-d*, *-r*, *-s*, or *-t* print only the
+  items having a description that matches the given regular expression.
+  Equivalent to *-Q --filter-pattern* 'regex'.
 
 *--status*::
-  Display  the  status of running instances of calcurse. If calcurse is
-  running, this will tell  if  the  interactive mode  was  launched  or  if
-  calcurse is running in background.  The process pid will also be indicated.
-
-*-t*[num], *--todo*[=num]::
-  Print the *todo* list and exit. If the optional number 'num' is given, then
-  only uncompleted todos having a priority equal to 'num' will be returned. The
-  priority number must be between 1 (highest) and 9 (lowest). It is also
-  possible to specify *0* for the priority, in which case only completed tasks
-  will be shown. Equivalent to *-Q --filter-type todo*, combined with
-  *--filter-priority* and *--filter-completed* or *--filter-uncompleted*.
-
-*--to* <date>::
-  Specify the end date of the range when used with *-Q*. Cannot be combined
-  with *--days*.
+  Display the status of running instances of calcurse, interactive or
+  background mode. The process pid is also printed.
+
+*-t*['num'], *--todo*[='num']::
+  Print the *todo* list. If the optional number 'num' is given, then only
+  uncompleted (open) todos having a priority equal to 'num' will be returned.
+  The priority number must be between +1+ (highest) and +9+ (lowest). It is
+  also possible to specify +0+ for the priority, in which case only completed
+  (closed)  tasks will be shown. Equivalent to *-Q --filter-type todo*,
+  combined with *--filter-priority* and *--filter-completed* or
+  *--filter-uncompleted*.
+
+*--to* 'date'::
+  Specify the end date of the day range when used with *-Q*. When used without
+  *--from* the start day is the current day. Cannot be combined with *--days*,
+  see <<_query,-Q --query>>.
 
 *-v*, *--version*::
-  Display *calcurse* version and exit.
+  Display *calcurse* version.
+
+*-x*['format'], *--export*[='format']::
+  Export user data in the specified format. Events, appointments and todos are
+  converted and echoed to stdout. Two formats are available: +ical+ and
+  +pcal+. The default 'format' is +ical+.
+
+[[_limiting_options]]
+.Filter, format and day range options
+*************************************
```

Comment:
Again, please make this a proper level-1 heading as it was before. While the new structure is semantically "correct", we should focus on making things easily visible in the man page. We should also avoid having many subheadings which make the man page harder to read.

Reply:
Corrected.